build.gradle.kts

```diff
@@ -171,6 +171,7 @@ val runIdeDirectory: String by project
 
 tasks.runIde {
     systemProperty("sonarlint.telemetry.disabled", "true")
+    //systemProperty("sonarlint.internal.sonarcloud.url", "https://sonarcloud.io/")
```

Comment:
Is there a reason to keep this comment?

Reply:
this is how you activate the custom URL. I will add a comment to explain that